docs/README.md

```diff
@@ -0,0 +1,44 @@
+# Why are we building OLM v1?
+
+Operator-lifecycle-manager's mission has been to manage the lifecycle of cluster extensions centrally and declaratively on Kubernetes clusters. Its purpose has always been to make installing, 
+running, and updating functional extensions to the cluster easy, safe, and reproducible for cluster administrators and PaaS administrators, throughout the lifecycle of the underlying cluster. 
+
+OLM so far has focused on providing unique support for these specific needs for a particular type of cluster extension, which have been coined as [operators](https://operatorhub.io/what-is-an-operator#:~:text=is%20an%20Operator-,What%20is%20an%20Operator%20after%20all%3F,or%20automation%20software%20like%20Ansible.). 
+Operators were classified as one or more Kubernetes controllers, shipping with one or more API extensions (CustomResourceDefinitions) to provide additional functionality to the cluster. 
+Over the last few years of running OLM in production cluster, it became apparent that there's an appetite to deviate from this coupling of CRDs and controllers, to encompass the lifecycling 
+of extentions that are not just operators.
+
+OLM has been assisting in defining a lifecycle for these extensions in which they get installed, potentially causing other extensions to be installed as well as dependencies, with a limited set of 
+customization of configuration at runtime, an upgrade model following a path defined by the extension developer, and eventual decommission and removal. There is a dependency model in which extensions can 
+rely on each other for required services that are out of scope of the primary purpose of an extension, allowing each extension to focus on a specific purpose. OLM also prevents conflicting 
+extensions from running on the cluster, either with conflicting dependency constraints or conflicts in ownership of services provided via APIs. Since cluster extensions need to be supported 
+with an enterprise-grade product lifecycle, there has been a growing need for allowing operator authors to limit installation and upgrade of their extension by specifing addtional environmental
+constraints as dependencies, primarily to align with what was tested by the operator author's QE processes. In other words, there is an ever growing ask for OLM to allow the author to enforce these 
+support limitations in the form of additional constraints specified by operator authors in their packaging for OLM.
+
+During their lifecycle on the cluster, OLM also manages the permissions and capabilities extensions have on the cluster as well as the permission and access tenants on the cluster have to the 
```

Comment:
This is a bit awkward, the sentence starts "During their lifecycle on the cluster," one would presume the next phrase would be begin with "extensions" (since that phrase is referencing extensions) and not OLM:
```
During their lifecycle on the cluster, extensions' cluster permissions and capabilities are managed  by OLM...
```
Or just cut that phrase and do something like:
```
OLM manages:
* extension permissions
* extension capabilities
* tenant's access to extensions
```
Or something along those lines.